app/thirdParty/googleRobotoFont-400-300-700.css

```diff
@@ -0,0 +1,168 @@
+/* cyrillic-ext */
+@font-face {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 300;
+  src: url(https://fonts.gstatic.com/s/roboto/v27/KFOlCnqEu92Fr1MmSU5fCRc4EsA.woff2) format('woff2');
```

Comment:
Hmm, these fonts are still loaded from google. If I really need google fonts, I usually install https://www.npmjs.com/package/@fontsource/roboto
But integrating that into the built might be some effort (idk), so I'll merge this as is for now